### PR TITLE
fix : README text start position

### LINKED ISSUE #8: Fix Readme text start position
The NanoPi Duo 2 readme have lines that are not aligned.

https://github.com/CyBear-Jinni/CBJ_Smart-Device_Resources/blob/master/devices_resources/nanopi_duo2/README.md

As you can see in the image the number 3 is forward to the command to execute "sudo ...".
And the first row is not in the same line as the last row.

Please make all the rows start in the same line.
If it is more beautiful you can make the scripts to run that have row to themselves (starts with sudo and nano setup_..)  little bit forward.

### PR DESCRIPTION
**Pull request by ghost**
fix : README text start position

I fix the README start position of the list, i using the HTML tag <p> for that, for README you can use many HTML tag.
* README

Resolves: #8

### CODE REVIEW

File: devices_resources/nanopi_duo2/README.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+<p>3. Open the terminal and connect to it with the following command:</p>
 
 `sudo screen /dev/ttyUSB0 115200`
 
 After that it will ask you to choose new password for root.
 
-4. Update the device date with the command 
+<p>4. Update the device date with the command </p>
 
 `sudo date -s "23 Sep 2020 16:01:50"`.
```
Comment (guyluz11) [inline devices_resources/nanopi_duo2/README.md]: Please remove the dot in the end.

**Review by guyluz11 — COMMENTED** [review]
This is better thanks.

For next time ask to take the issue before creating pull request for it.

Please leave a comment asking to take it in the issue so I will be able to assigne it to you.

**Comment by ghost** [comment]
Excuse me i'm new in pull request, i correct that

**Comment by guyluz11** [comment]
@72y9op thanks for the fix.

I am closing this pull request as it is currently to `master` branch, and we do not push straight to `master`.

Please open new pull request to `dev` and I will merge it.

If you need help with that you can join the discord and I will help you with that


[](https://discord.gg/mUXfwUY)

**Comment by guyluz11** [comment]
And leave a comment on [the issue](https://github.com/CyBear-Jinni/CBJ_Smart-Device_Resources/issues/8), I need to assign it to you.

**Pull request closed without merging**